ios/FluentUI.Demo/FluentUI.Demo/Demos/LeftNavDemoController.swift

```diff
@@ -206,48 +205,33 @@ class LeftNavMenuViewController: UIViewController {
 
     private var leftNavMenuList = MSFList(sections: [])
 
-    private var leftNavAccountLabelsView: UIView {
-        let nameLabel = UILabel()
-        nameLabel.translatesAutoresizingMaskIntoConstraints = false
-        nameLabel.font = UIFont.preferredFont(forTextStyle: .subheadline)
-        nameLabel.text = "Kat Larrson"
-
-        let titleLabel = UILabel()
-        titleLabel.translatesAutoresizingMaskIntoConstraints = false
-        titleLabel.font = UIFont.preferredFont(forTextStyle: .caption2)
-        titleLabel.textColor = .darkGray
-        titleLabel.text = "Designer"
+    private var leftNavAccountView: UIView {
+        leftNavAvatar.state.presence = .available
+        leftNavAvatar.state.primaryText = "Kat Larrson"
+        leftNavAvatar.state.secondaryText = "Designer"
+        leftNavAvatar.state.image = UIImage(named: "avatar_kat_larsson")
 
-        let labelsStackView = UIStackView(arrangedSubviews: [nameLabel, titleLabel])
-        labelsStackView.translatesAutoresizingMaskIntoConstraints = false
-        labelsStackView.axis = .vertical
-        labelsStackView.spacing = Constant.verticalSpacing
+        let chevron = UIImageView(image: UIImage(named: "ic_fluent_ios_chevron_right_20_filled"))
+        chevron.tintColor = Colors.textPrimary
 
-        return labelsStackView
-    }
+        let persona = MSFPersonaView()
+        persona.state.persona = leftNavAvatar.state
+        persona.state.titleTrailingAccessoryView = chevron
 
-    private var leftNavAccountView: UIView {
-        let avatarView = leftNavAvatar.view
-        avatarView.translatesAutoresizingMaskIntoConstraints = false
-        let labelsView = leftNavAccountLabelsView
+        let personaView = UIStackView(arrangedSubviews: [persona.view])
```

Comment:
The stackView is not working very well here.
The persona.view should just be constrained the same way the List is setup.

![Screen Shot 2021-04-07 at 9 58 21 AM](https://user-images.githubusercontent.com/68076145/113905433-ffa03280-9787-11eb-904a-a7544a6b2ee6.png)

Reply:
I think this might be a SwiftUI issue, will update you offline about it